docs/language/operators/summarize.md

```diff
@@ -5,14 +5,23 @@
 ### Synopsis
 
 ```
-[summarize] [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>] ...]
+[summarize] [<field>:=]<agg>
+[summarize] [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...]
+[summarize] [<field>:=]<agg> [by [<field>][:=<expr>][, [<field>][:=<expr>]] ...]
+[summarize] [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>][, [<field>][:=<expr>]] ...]
+[summarize] by [<field>][:=<expr>][, [<field>][:=<expr>] ...]
```

Comment:
After seeing the first iteration of this PR, @nwt correctly pointed out that the expanded/corrected original synopsis line was way too long and complex. His first thought on how to improve it was to break it out into more lines, so that's what I've done on this next iteration and indeed I'm liking it more. I like leading with the shorter one that represents a very common use, then builds up to the full/beastly one, so if a user is learning incrementally hopefully they'll get the idea.